docs/FrameDescriptions.md

```diff
@@ -1,21 +1,121 @@
-Following are the descriptions of the Frames used by different devices in our framework following these issues:
+**Following you can find the descriptions of the frames used by different devices in teleoperation framework:**
 
-**Oculus Device**
+## Inertial Frame
+
+We define the inertial frame in the human side as following:
+
+<p align="center">
+  <img width="300" src="./images/InertialFrame.jpg">
+</p>
+
+## Oculus Device
+According to the [Oculus documentation](https://developer.oculus.com/documentation/native/pc/dg-sensor/), the frames attached to the Oculus Virtual Reality (OVR) headset are as following:
+```
+The x-z plane is aligned with the ground regardless of camera orientation.
+As seen from the diagram, the coordinate system uses the following axis definitions:
+- Y is positive in the up direction.
+- X is positive to the right.
+- Z is positive heading backwards.
+Rotation is maintained as a unit quaternion, but can also be reported in yaw-pitch-roll form. Positive rotation is counter-clockwise.
+```
+However, the custom they have used for defining the euler angles and their composition order are different from ours.
+[Following line of code](https://github.com/robotology/walking-teleoperation/blob/master/modules/Oculus_module/src/OculusModule.cpp#L386-L389
+) provides us the rotation from [inertial frame](#inertial-frame) (attached to the human head at the begning) to the oculus headset at timestep `t` expressed in preferred coordinate system:
+
+```
+iDynTree::toEigen(m_oculusRoot_T_headOculus).block(0, 0, 3, 3)
+                = iDynTree::toEigen(iDynTree::Rotation::RPY(-desiredHeadOrientationVector(1),
+                                                            desiredHeadOrientationVector(0),
+                                                            desiredHeadOrientationVector(2)));
+```                                                           
 
 <p align="center">
   <img width="500" height="500" src="./images/Oculus.png">
 </p>
 
-**Virtualizer (treadmill)**
+## treadmill (Virtualizer Cyberith)
+
+According to the measurement information we acquire from the cyberith treadmill, we identify the frame of Virtualizer as following:
```

Comment:
I think that the image at this stage is a bit confusing because it is not clear what the frame of the Virtualizer is. Down below you explained well how the rotation of the virtualizer is obtained. I would suggest moving this image after the explanation.